test/svg-sprite/config.test.js

```diff
@@ -24,3 +24,61 @@ describe('testing constructor', () => {
         });
     });
 });
+
+describe('testing filter', () => {
+    let config;
+    const modes = ['stack', 'defs', 'view', 'css', 'symbol'];
+
+    beforeEach(() => {
+        config = new SVGSpriterConfig({});
+    });
+
+    it('should return empty object if empty object passed', () => {
+        expect.hasAssertions();
+        expect(config.filter({})).toStrictEqual({});
+    });
+
+    it.each(modes)('should filter plainObjects and set %p mode if mode is not specified', mode => {
```

Comment:
What's this `%p`? First time I see notice it :)

Reply:
NVM, just saw the docs https://jestjs.io/docs/api#describeeachtablename-fn-timeout